From 7af99168387dd3fec3408835b40bd84a9b191271 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 19 Apr 2012 16:41:04 +0200
Subject: [PATCH] network/parse-ip-opts.sh: remove check for netroot

with anaconda and all other kind of stuff, we might want network, even
for root not on the network
---
 modules.d/40network/parse-ip-opts.sh |    8 --------
 1 file changed, 8 deletions(-)

diff --git a/modules.d/40network/parse-ip-opts.sh b/modules.d/40network/parse-ip-opts.sh
index 97702a2..5637a9a 100755
--- a/modules.d/40network/parse-ip-opts.sh
+++ b/modules.d/40network/parse-ip-opts.sh
@@ -17,14 +17,6 @@
 command -v getarg >/dev/null          || . /lib/dracut-lib.sh
 command -v ibft_to_cmdline >/dev/null || . /lib/net-lib.sh
 
-# Check if ip= lines should be used
-if getarg ip= >/dev/null ; then
-    if [ -z "$netroot" ] ; then
-        echo "Warning: No netboot configured, ignoring ip= lines"
-        return;
-    fi
-fi
-
 # Don't mix BOOTIF=macaddr from pxelinux and ip= lines
 getarg ip= >/dev/null && getarg BOOTIF= >/dev/null && \
     die "Mixing BOOTIF and ip= lines is dangerous"
